### DISCUSSION TITLE
Menubar Icon size, iCloud & Reminders Integration

### DISCUSSION THREAD

**Discussion by pzeepzee** [rampatra/todobar-community#2] [Ideas]
Started using the app and love it quite a bit, but feel there are things that can be improved.
1.  First is the icon size in the menubar, the size of the icon for unfinished tasks in the menubar is so tiny, I have to squint. Can you make it bigger?
2. I will also like for the app to sync tasks across laptops via iCloud?
3. Finally, can it be integrated with Reminders please?
Thanks

**Comment by rampatra** [comment]
@pzeepzee Thanks for your expounded feedback. I am already working on the first two points raised here. The third one regarding Reminders integration, can you kindly elaborate a bit more? Why do you need this?

**Reply by pzeepzee** [reply]
Thanks Rampatra, glad to hear you are working on the first and second requests, as per integration with Reminders, it extends the apps usability, as i can manage tasks from my iPhone/iPad without you necessarily creating an iOS app. I also would like to have a history of tasks not only reside in your app, which is great already, but in Apple's Reminders as well, thanks

**Reply by zenminimalist** [reply]
Any news on the Apple Reminders integration? That would b an instant buy for many i think.
